docs-v2/reference/sdks/node.mdx

```diff
@@ -459,6 +459,50 @@ await nango.setMetadata('<INTEGRATION-ID>', 'CONNECTION-ID', { 'CUSTOM_KEY1': 'C
 **Response**
 
 Empty response.
+<Expandable>
```

Comment:
Same thing, maybe this can be part of the docs of the existing method?

Reply:
See above comment